src/test/ui/liveness/liveness-forgot-ret.stderr

```diff
@@ -1,8 +1,10 @@
 error[E0308]: mismatched types
-  --> $DIR/liveness-forgot-ret.rs:3:25
+  --> $DIR/liveness-forgot-ret.rs:3:19
    |
 LL | fn f(a: isize) -> isize { if god_exists(a) { return 5; }; }
-   |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected isize, found ()
+   |    -              ^^^^^ expected isize, found ()    - expected because of this statement
```

Comment:
`expected because of this statement` doesn't seem right: it's expected because of the return type, not because of the return statement.

Reply:
You're right. This was introduced in a different recent PR. I haven't had the time to dig into it yet, but that is a definitively misleading label. I'd like to work on it on a separate PR.